rand_chacha/src/guts.rs

```diff
@@ -28,6 +28,19 @@ pub struct ChaCha {
     pub(crate) d: vec128_storage,
 }
 
+// Custom PartialEq implementation as `vec128_storage` doesn't implement it.
+impl ::core::cmp::PartialEq for ChaCha {
```

Comment:
This seems pretty non-ideal, but I'm hoping to not have to also try to get this change into `vec128_storage`.

Reply:
Nothing wrong with a custom implementation, but that `unsafe` code — we've been trying to minimise usage, so I'm not keen on this.

Reply:
You could ask @kazcw about that, but [looks to me](https://github.com/cryptocorrosion/cryptocorrosion/blob/master/utils-simd/ppv-lite86/src/generic.rs#L19) like you can just do `let b: [u32; 4] = self.b.into();`.

Reply:
My previous version used `Into<[u32; 4]>` but it's not implemented on all platforms.

Reply:
I see this code was added since `ppv-lite` 0.2.6. @kazcw could you release a new version please?

I don't wish to add this unsafe code, which makes us blocked on `ppv-lite`.

Reply:
@dhardy Sure, I can do that this weekend.